mynah-ui/package.json

```diff
@@ -12,7 +12,7 @@
         "lintfix": "eslint -c .eslintrc.js --fix --ext .ts ."
     },
     "dependencies": {
-        "@aws/mynah-ui-chat": "npm:@aws/mynah-ui@3.2.9",
+        "@aws/mynah-ui-chat": "npm:@aws/mynah-ui@4.0.1",
```

Comment:
I'm looking at the commits since there doesn't seem to be release notes -> https://github.com/aws/mynah-ui/commits/main/